valama.vlp

```diff
@@ -519,4 +527,5 @@
 		<language id="fr"/>
 	</gettext>
 	<gladeui id="101" filename="src/Ui/Editors/Editor_Gettext.glade"/>
+	<valasource id="102" filename="buildsystems/config.vapi"/>
```

Comment:
The config.vapi is generated automatically, the user is not supposed to access it. So, adding it to the CodeContext behind the scenes and not showing it as a source file actually is the right thing to do.